### ISSUE TITLE
Command line/ Continuous Integration API?

### ISSUE THREAD

**Issue by mhpaler** [codeulike/SandboxBerry#1]
It appears it's current incarnation requires points and clicks on a windows box. Is that correct?

Ideally a tool like this could be run from a command line so it could potentially be integrated into a proper continuous integration system.  Is this something that is possible with this tool?

**Comment by codeulike** [comment]
Yep the app is point and click, but in terms of code the UI is well separated from the code that actually does stuff. SandboxBerryLib is where all the work is actually done, it would be pretty easy to add a command line or Powershell wrapper. I'll bear it in mind ....
